### DISCUSSION TITLE
AI強大繪畫能力!

### DISCUSSION THREAD

**Discussion by Timmy110403540** [1101-GS3073-C/Python-Community-GS3073-C#129] [General]
NVIDIA藉著 NVIDIA Research 的 GauGAN2 協助，成功讓使用者在輸入幾行字後，浮現出逼真的畫面。
舉個簡單例子；當輸入: "a sun"，一個太陽就會出現。但若使用者輸入更多細節，像是:"a man enjoys the sunshine at a beach"，那就會出現一個男人在沙灘上享受著陽光的洗禮。越是細膩的描述，越能精確地呈現!
而目前，NVIDIA Selene 已使用 1 千萬張高品質風景圖片訓練 GauGAN2 AI 模型，並且使用的是NVIDIA DGX SuperPOD 系統。
另外，也使用分割映射技術，混合使用繪畫及文字達到寫實的效果。
這項超酷的技術，真是造福了不擅於畫畫的人(當然包括我🤣)。也期待這項科技未來的發展，能開創出更多的功能!

文章來源:https://technews.tw/2021/11/25/nvidia-showcases-gaugan2s-artificial-intelligence-painting-capabilities/

**Comment by Jay-722** [comment]
太神啦，這可以滿足不會畫畫的人體驗自己做出漂亮的畫作。

**Reply by pocper1** [reply]
之後美術課知道該用甚麼來畫圖了吧XD

**Comment by bonny123865** [comment]
喔買尬，這樣以後就可以真的實現想像力就是你的超能力？！
而且居然用 1 千萬張風景圖訓練模型，超級多的！

**Comment by wh20502** [comment]
我有點好奇，它畫出來的人跟太陽是人類手動輸入的嗎(資料庫)，如果是靠訓練來的話，搞不好以後會有ai抽象藝術大師。

**Comment by minniewei** [comment]
有些人 是有想法但畫不出來，這項技術能讓他們發揮所長!!

**Comment by ayakasanma** [comment]
其實這也不能算創作（？
用AI作畫是將資料庫有關於本主題的畫作都提取出來然後用大數據去繪製。相信畫家這個職業還是會存在的，不過這項服務的確造福很多不會畫圖的手殘黨XD

**Reply by cobb110202008** [reply]
認同，畫家創作的畫是獨一無二的，AI仍然無法取代畫家的個人風格及創意。

**Comment by ghost** [comment]
可是這樣子如果不同人輸入同樣的字詞，跑出來的畫作是否會相同呢?又或是不同?

**Comment by Shiba010** [comment]
太厲害了吧！感覺他可以逼真的模擬出各種人類能想像到的場景。

**Comment by 110409538** [comment]
這也太酷了吧！但是少了自己畫畫的樂趣和成就感。

**Comment by jack90325** [comment]
那感覺設計師這個職位要被挑戰了，又有被替代的可能

**Comment by Lynn12378** [comment]
太強了吧，我也想向AI一樣這麼厲害
